api-client/src/test/java/com/xing/api/data/SafeCalendarTest.java

```diff
@@ -29,4 +29,13 @@ public void testSet() throws Exception {
         assertThat(calendar.get(Calendar.HOUR)).isEqualTo(0);
         assertThat(calendar.get(Calendar.YEAR)).isEqualTo(1970);
     }
+
+    @Test
+    public void testIsEmpty() throws Exception {
+        SafeCalendar notEmtyDate = new SafeCalendar(2016, 5, 15);
```

Comment:
Can you change this with "2017, 1st January" and put a comment as "Default value for calendar" so that we know even if it has default value it can distinguish between empty and set value :)